great_expectations_cloud/agent/agent.py

```diff
@@ -299,6 +299,7 @@ def _handle_event_as_thread_exit(
                     success=False,
                     created_resources=[],
                     error_stack_trace="The version of the GX Agent you are using does not support this functionality. Please upgrade to the most recent image tagged with `stable`.",
+                    processed_by="runner" if self._config.queue == "gx-runner" else "agent",
```

Comment:
This seemed like the easiest way to accomplish adding `processed_by` here. I'm open to other ideas, though, especially if we don't want the agent to know about the runner queue.